Validation/RecoTau/src/TauTagValidation.cc

```diff
@@ -162,14 +163,14 @@ void TauTagValidation::bookHistograms(DQMStore::IBooker & ibooker, edm::Run cons
   pileupTauVisibleMap.insert( std::make_pair(TauProducer_+"Matched" ,pileupTemp));
 
   int j = 0;
-  for ( std::vector< edm::ParameterSet >::iterator it = discriminators_.begin(); it!= discriminators_.end();  it++, j++)
+  for(auto&& it : discriminators_)
```

Comment:
Const reference (const auto&) should be enough here, no? In my mind even previously const iterator will be better than non const one.